.travis.yml

```diff
@@ -1,3 +1,6 @@
+branches:
```

Comment:
Use whitelist instead:
```yaml
# safelist
branches:
  only:
  - master
```

maybe put in something like `'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit; fi'`